index.js

```diff
@@ -192,8 +192,12 @@ module.exports = {
 
         // sanitise the mime argument
         mime = String(mime).toLowerCase().replace(/\s/g, E).replace(/^([^;]+).*$/g, '$1');
-        var result = db[mime] || module.exports.guess(mime);
-
+        var result;
+        if(db[mime]){
```

Comment:
Here, re-assign the result with:
`result = result ? {...result} : module.exports.guess(mime)`

---
Because `db[mime]` lookup is happening twice unnecessarily.